### ISSUE TITLE
failed action due to matrixStats

### ISSUE THREAD

**Issue by avallecam** [Joskerus/epimodelac#8]
After solving #7 by removing sandpaper and pegboard from `renv.lock` we got this new failed action (follow the [link to the error message](https://github.com/Joskerus/Nuevoformato/actions/runs/7023861402/job/19111336225))

The error line is the following ([reference](https://github.com/Joskerus/Nuevoformato/actions/runs/7023861402/job/19111336225#step:6:2564))

```
  - Installing matrixStats ...                    FAILED
```

**Comment by avallecam** [comment]
> Thank you, Hugo.
> 
> Yes, indeed. It was due to the version included in the `renv.lock`. Yesterday we identified that {MatrixStats} was part of {EpiEstim} dependency tree (within {coarseDataTools}).
> 
> We solved it in [d07d351](https://github.com/Joskerus/Nuevoformato/commit/d07d3517bf4e80bb894f2019de8f09ea908d6959) and [421eda6](https://github.com/Joskerus/Nuevoformato/commit/421eda63a03ff1dfec6905ba74e8293f8f41fd2d) by replacing the `renv.lock` with the one I generated in the workbench-spanish repo.
> 
> I replaced the explicit installation from r-universe to CRAN with `sandpaper::pin_version()`, removing the explicit reference to all the dependency tree in the `renv.lock` file, leaving to the GitHub action to install them, see this commit [avallecam/workbench-spanish@8408159](https://github.com/avallecam/workbench-spanish/commit/84081595ccfd0878ef4112f8eb7bf0866d59b472)
> 
> Then I tried the same for {rstan} which also generated a similar error in previous workflows, so added a pin_version of it too in a later commit, and it worked fine.
> 
> This solution is still under observation.

Originally posted by @avallecam in https://github.com/Joskerus/Nuevoformato/pull/6#issuecomment-1831668112

**Comment by avallecam** [comment]
When using `sandpaper::pin_version()` I used the [`renv::record()`](https://rstudio.github.io/renv/reference/record.html#ref-examples) notation as suggested in [pin_version](https://carpentries.github.io/sandpaper/reference/dependency_management.html#arguments) and as in these lines of code:

```
record <- list(Package = "EpiEstim", Version = "2.2-4", Source = "Repository", Repository = "CRAN")
sandpaper::pin_version(list(EpiEstim = record))
```

``` 
record <- list(Package = "rstan", Version = "2.32.3", Source = "Repository", Repository = "CRAN")
sandpaper::pin_version(list(rstan = record))
```
